webrender/src/layer.rs

```diff
@@ -6,6 +6,7 @@ use euclid::{Matrix4D, Point2D, Rect, Size2D};
 use spring::{DAMPING, STIFFNESS, Spring};
 use webrender_traits::{PipelineId, ScrollLayerId};
 
+/// Contains scroll and transform information for scrollable and root stacking contexts.
 pub struct Layer {
     // Manages scrolling offset, overscroll state etc.
```

Comment:
These will need to be fixed too, although not necessarily in your PR since you didn't touch them.